src/main/java/de/retest/recheck/RemoteRecheckImpl.java

```diff
@@ -0,0 +1,48 @@
+package de.retest.recheck;
+
+import java.util.prefs.Preferences;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.recheck.auth.BrowserAuthenticationHandler;
+import de.retest.recheck.auth.RetestAuthentication;
+import de.retest.recheck.persistence.CloudPersistence;
+
+public class RemoteRecheckImpl extends RecheckImpl {
```

Comment:
@roesslerj any objections?

Reply:
Hmm. I am not sure about that. Remote is IMHO more directly understandable, Rehub requires extra knowledge. What would be your main motivation for using that name?

Reply:
Plain marketing. I agree that `RemoteRecheckImpl` is easier to understand, but also the term "recheck" in `RecheckImpl` requires extra knowledge, and so does `Eyes` from Applitools or `Percy` from Percy.

I though this may be a good way to incorporate our product name(s)? Otherwise "rehub" is mentioned nowhere, only once in a log.

Reply:
Nach kurzer TelCo: Besser keine separate Klasse mit Options und Property.